### ISSUE TITLE
Provide captions for pre-recorded audio content

### ISSUE THREAD

**Issue by adamjarling** [avalonmediasystem/structured-metadata-editor-react#68]
Description
SME audit report line item 1.2.2, referenced in this issue: #64

WCAG 2.1 Captions (Pre-recorded). Provide captions for pre-recorded audio content.

This one could be a biggie.

### Done Looks Like
- Captions are exposed to the front-end via an application endpoint URL.
- This audit test passes in whichever tool described the failure.

### To Do
- [ ] Are we currently generating captions, if an audio file contains speech?   What happens if there is no speech, just a generic "Music" caption?
- [ ] If we're not generating captions, discuss how that fits into this scenario.
- [ ] If we are generating captions, work to expose those alongside an item, where the front end can grab them.
- [ ] If captions are exposed, grab on the front end, and figure out how to display them in.  This will be broken out into its own issue, where we'll create a `<Captions />` component, or JS module to isolate this logic in our application.
- [ ] Re-test.
